src/hooks/useMediaQuery/useMediaQuery.ts

```diff
@@ -1,17 +1,24 @@
-import React from "react";
+import { useLayoutEffect, useState } from "react";
+import { getWindow } from "../../__internal__/dom/globals";
 
-export default function useMediaQuery(queryInput: string): boolean {
-  const query = queryInput.replace(/^@media( ?)/m, "");
+export default function useMediaQuery(queryInput: string): boolean | undefined {
+  const query = queryInput.replace(/^@media\s*/i, "");
```

Comment:
comment: this regex pattern will match differently as it's now not case-sensitive for example